### DISCUSSION TITLE
Traditional Chinese Translate Support

### DISCUSSION THREAD

**Discussion by xMikux** [pelican-dev/panel#199] [Ideas]
As pelican already support Simplified Chinese, would be great to support Traditional Chinese translate!

**Comment by DerLev** [comment]
Translations are done here: https://crowdin.com/project/pelican-dev
Although you will need to contact one of the Crowdin Project Managers and ask them to add Traditional Chinese to the list of translations.
Hope this helps

**Comment by iamkubi** [comment]
We're only able to support 2 letter language codes at this time which unfortunately means we're only able to have a single dialect of languages that have multiple.
